Guesser/AbstractTypeGuesser.php

```diff
@@ -11,9 +11,9 @@
 
 namespace Sonata\PropelAdminBundle\Guesser;
 
+use Propel\Bundle\PropelBundle\Form\PropelTypeGuesser;
 use Sonata\AdminBundle\Guesser\TypeGuesserInterface;
 use Sonata\AdminBundle\Model\ModelManagerInterface;
-use Symfony\Bridge\Propel1\Form\PropelTypeGuesser;
```

Comment:
Ok so looking at [this issue](https://github.com/symfony/symfony/issues/13091), I can understand why.

> It means also we should drop SF obsolete versions.

That's the plan for every next major release of the sonata bundle (I think the requirement might even be sf 2.8).

Now regarding this issue we had previously, namely [your tweet](https://twitter.com/heristop/status/771746480742891521), and your previous [comment](https://github.com/sonata-project/SonataPropelAdminBundle/pull/71#discussion_r77372796), I think I understand why you reacting like this. You are taking some time to help us out with this bundle, and on behalf of all the Sonata team, I thank you for it, because I know what how time-consuming it can be to contribute something. That is maybe your way of giving back to this piece of open-source software, to show your gratefulness for the time it might have saved you.

With that in mind, I can understand how my many comments can be overwhelming, and may make me sound very distrustful and ungrateful to you. When merging, we do not do a background check on you to evaluate if you know how to code. Instead, we just try to read the code and ask questions until we fully understand it. We also try to make sure that **anyone** that reads your PR or better, your commits, can understand *why* you did every change. The thing is, when we click the merge button, the code you wrote becomes ours to maintain.

Now when we do not do that, when we merge blindly because "it looks good", [here is what happens](https://github.com/sonata-project/SonataMediaBundle/pull/909#discussion_r77162504). As you can see in this discussion, we get [issues](https://github.com/sonata-project/SonataMediaBundle/issues/1065) that are very hard to understand, and when we finally find the guy that introduced the faulty piece of code, what do we hear? ["I don't have time to look at this now. Since it's FOSS software, feel free to contribute. ;)"](https://github.com/sonata-project/SonataMediaBundle/issues/1065). So basically, this is our problem now. This guy made his PR to get his particular application to work, and when we ask for help, we consider ourselves lucky that he even answers to us (not everybody does that).

I guess you must be asking yourself some questions now. "How is it possible that no one knows how this work ?". Well that's wrong, there are people like @rande who do know how most of all this works. However, they are busy with their lives, or trying to keep up with the fuckton of other comments they have to review. Your contribution is maybe very nice, but if you get a review from him, you will be very lucky, and maybe your contribution will get merged fast because he fully understands it.

To spare you the frustration of feeling ignored, I come and try to understand your contribution. But since I do not want to be held accountable for bugs or problems I might have created by merging, I have no choice but to ask you some questions, that will help me understand both the PR and the software. I also am a grammar-nazi, very nitpicky about the commits and every single semicolon in your PR. I'll ask for explanation, tests and docs. While this might not be very useful when dealing with a small application, I deeply believe it is important for an open-source project. This makes of course the PR harder to merge **but** the comments I make, you will get them quickly. My first comment on this PR was literally 3 minutes after you made your PR. Do you know many project where this happens? Also, I try to make my comments *actionable*. Remove an alias? Easy-peasy! Changing a tilde to a caret? Piece of cake. Explaining what you did clearly? Shouldn't be a problem, and should be done in your commit messages (yes, I am using the plural here, not everything needs to be in one huge commit that says "fix bug").

So with all that in mind, I hope you are not discouraged, and I renew my commitment into giving you fast feedback, and all the help I can provide you with making this a no-brainer to merge. And you are overwhelmed and do not want to finish the job, I might even go as far as reworking your commits and doing it for you.